Installing-Oppia-(Linux;-Python-3).md

```diff
@@ -0,0 +1,150 @@
+**Note:** If you just want to create and share explorations, you may be able to use the hosted server at https://www.oppia.org (in which case you don't need to install anything).
+
+*These installation instructions were last tested on 24 July 2021. For more information on issues that may occasionally arise with the installation process, please contact _vojtech.jelinek@hey.com_ or see the [Troubleshooting](https://github.com/oppia/oppia/wiki/Troubleshooting) page (but that page is not yet updated for Python 3).*
+
+## Downloading Oppia ##
+
+1. Create a new, empty folder called `oppia/` in your computer's home folder. If you already use `oppia/` for Python 2 Oppia feel free to use some different folder name. 
+
+1. Navigate to it (`cd oppia`), then [fork and clone](https://help.github.com/articles/fork-a-repo/) ([why?](https://github.com/oppia/oppia/wiki/Why-fork-and-clone-Oppia%3F)) the Oppia repo so that it gets downloaded into `oppia/oppia`. 
+
+1. Navigate to this folder (`cd oppia`) and run `git remote add upstream https://github.com/oppia/oppia.git` (or, if you're using SSH, use `git@github.com:oppia/oppia.git`) to add an `upstream` remote that's linked to Oppia's main repository. (This will be necessary when submitting PRs later.)
+
+1. **Run `git pull https://github.com/brianrodri/oppia.git py3-migration` to pull the changes from the Python 3 PR.**
+
+
+## Prerequisites ##
+
+_The following instructions will install Oppia on your local machine._
+
+Oppia relies on a number of programs and third-party libraries. Many of these libraries are downloaded automatically for you when you first run the `start.py` script provided with Oppia (see step 1 in the next section). However, there are some things that you will need to do beforehand:
+
+1. Make sure you have curl (used to download third-party libraries), setuptools (needed for installing coverage, which checks test coverage for the Python code), git (which allows you to store the source in version control), python-dev (which is used for the numpy installation), python-pip (which is also used for the numpy installation) and pyyaml (which is used to parse YAML files):
+
+  ```
+    sudo apt-get install curl openjdk-8-jre python3-setuptools git python3-dev python3-pip python3-yaml unzip python-matplotlib python3-matplotlib
+  ```
+
+Alternatively, if you are on Debian/Ubuntu, you can use the `install_prerequisites.sh` script to install these. From the oppia directory:
+
+```
+bash scripts/install_prerequisites.sh
+```
+
+2. We heavily recommend usage of virtual environment for working with Oppia. Here is a short guide for using [direnv](https://direnv.net/):
```

Comment:
I think "highly recommend" is more common.

Reply:
This file change isn't actually part of this PR. I just forgot to push the latest `master` from `oppia/oppia` to the `master` of `U8NWXD/oppia-wiki`. I've fixed this now